packages/react/ds/src/score-select/score-select.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  ButtonGroup,
+  ButtonGroupItem,
+} from '../button/button-group/button-group.js';
+import { FormField } from '../forms/form-field.js';
+import { ScoreSelectProps } from './type.js';
+
+export const ScoreSelect: React.FC<ScoreSelectProps> = ({
+  name,
+  size = 'medium',
+  options,
+  defaultValue,
+  label,
+  hint,
+  leftLabel,
+  middleLabel,
+  rightLabel,
+  onChange,
+}) => {
+  return (
+    <FormField
+      className="gi-w-full"
+      label={{ text: label }}
+      hint={hint ? { text: hint } : undefined}
+    >
+      <div className="gi-score-select-button-group">
+        <ButtonGroup
```

Comment:
it is missing some aria attributes like ```role, aria-label```..
would be good to include some. the same for ButtonGroupItem